### ISSUE TITLE
Unable to reinstall 

### ISSUE THREAD

**Issue by Marcus235** [ThaYoung1/com.thayoung1.strava#19]
I’m trying to reinstall Strava again on my Homey Pro but keep getting fault 
![IMG_3686](https://github.com/user-attachments/assets/087c69b8-3db6-4da0-85d3-f859110bc0fe)

**Comment by ThaYoung1** [comment]
Hi @Marcus235, 

You don't need to use the application settings page. You need to add a new device in Homey and follow the steps that are shown in that wizard.

It's somewhat confusing, therefore I will remove the settings page in the near future.

Does this help solving the problem you are experiencing?

**Comment by Marcus235** [comment]
Yes thanks! Vriendelijke groet, Marcus BitterlingOp 2 jan 2025 om 12:27 heeft ThaYoung1 ***@***.***> het volgende geschreven:﻿
Hi @Marcus235,
You don't need to use the application settings page. You need to add a new device in Homey and follow the steps that are shown in that wizard.
It's somewhat confusing, therefore I will remove the settings page in the near future.
Does this help solving the problem you are experiencing?
